notary/content.md

```diff
@@ -0,0 +1,82 @@
+%%LOGO%%
```

Comment:
For this one to work, you'll need to add either `notary/logo.png` or `notary/logo.svg` BTW :smile: